app/client/cli/utils.go

```diff
@@ -190,3 +190,13 @@ func attachPwdFlagToSubcommands() []cmdOption {
 		c.Flags().StringVar(&pwd, "pwd", "", "passphrase used by the cmd, non empty usage bypass interactive prompt")
 	}}
 }
+
+func unableToConnectToRpc(err error) error {
+	fmt.Printf("❌ Unable to connect to the RPC @ \033[1m%s\033[0m\n\nError: %s", remoteCLIURL, err)
+	return nil
+}
+
+func rpcResponseCodeUnhealthy(statusCode int, response []byte) error {
+	fmt.Printf("❌ RPC reporting unhealthy status HTTP %d @ \033[1m%s\033[0m\n\n%s", statusCode, remoteCLIURL, response)
```

Comment:
This might be overkill so consider it optional

```go
func formatText(s string) string {
   return  fmt.Printf("\033[1m%s\033[0m", s)
}
```

Reply:
If it's not clear like I understand from https://github.com/pokt-network/pocket/pull/169#discussion_r1016924253

it means that something like this:

https://github.com/pokt-network/pocket/pull/169/commits/f2c765fa18a3e0fd02134907db7f537dc1fc591e

is not overkill and it's better than a comment.